crates/viewer/re_renderer/src/video/decoder/web.rs

```diff
@@ -86,14 +94,20 @@ impl VideoDecoder {
         data: Arc<re_video::VideoData>,
     ) -> Result<Self, DecodingError> {
         let frames = Arc::new(Mutex::new(Vec::with_capacity(16)));
+        let timescale = data.timescale;
 
         let decoder = init_video_decoder({
             let frames = frames.clone();
             move |frame: web_sys::VideoFrame| {
-                frames.lock().push((
-                    TimeMs::new(frame.timestamp().unwrap_or(0.0)),
-                    VideoFrame(frame),
-                ));
+                let composition_timestamp =
+                    Time::from_micros(frame.timestamp().unwrap_or(0.0), timescale);
```

Comment:
this was `TimeMs::new(frame.timestamp().unwrap_or(0.0)),` before but now we interpret the `timestamp` as microseconds where we assumed milliseconds before. That looks like a bug!

Reply:
That's intentional, the value returned here is the same as the one passed in when we create the `EncodedVideoChunk`. We want to match `video-decode-test` behavior more closely (because it Just Works :tm:), and it uses microseconds as well, so I switched to using microseconds here:

https://github.com/rerun-io/video-decode-test/blob/885bad29aa27f0ed87560696cc8dec953274d6b9/index.js#L269-L270

Reply:
Note that the `VideoFrame` API guarantees that the values passed in are the same as the ones passed out

https://w3c.github.io/webcodecs/#output-videoframes @ 1. 1.